contracts/land/ILANDRegistry.sol

```diff
@@ -59,4 +65,15 @@ interface ILANDRegistry {
     address indexed _caller,
     address indexed _deployer
   );
+
+  event Ping(
+    address indexed _caller,
+    address indexed _target
+  );
+
+  event Reactivate(
+    uint256 indexed _assetId,
+    address indexed _newOwner
+  );
+
```

Comment:
What do you think of adding an event when someone set a new ReactivateAuthorized?

```solidity
event ReactivateAuthorized(address indexed _caller, address indexed _operator, bool _authorized)
```